mattumland/next-graphcms-shows
# Add and implement utility function to validate artist urls

## What issue does this solve?
https://codevalapp.github.io/next-graphcms-shows/issues/4/

## Background
We can't assume everyone will enter the full url with protocols in the database so we need to catch invalid urls that are returned from graphCMS. This adds a utility function that double checks if links contains `https` or `http` protocols and adds `https` if the link is missing it. 

## Manual testing instructions
1. Boot up dev server
2. Go to `shows`
3. Click on multiple `Mama Said 30th Anniversary`, confirm the `website` link works. 
4. Double check a few more websites on other show pages.
